redisearch/query.py

```diff
@@ -51,6 +51,48 @@ def return_fields(self, *fields):
         self._return_fields = fields
         return self
 
+    def summarize(self, *fields, **options):
```

Comment:
I prefer that the kwargs will have the defaults already. I hate it when python code sends you on an RTFM mission to decipher what **kwargs actually mean...

Reply:
The problem with this is that it disallows me from specifying multiple fields, and the user must then do one field at a time. I guess it's acceptable, but annoying.